src/main/resources/schemas/exchange/organizationDirectory.xsd

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+           xmlns:tns="urn:gov:hhs:fha:nhinc:exchange:directory"
+           targetNamespace="urn:gov:hhs:fha:nhinc:exchange:directory"
+           elementFormDefault="qualified">
+
+  <xsd:complexType name="organizationType">
+    <xsd:sequence>
+      <xsd:element name="name" type="xsd:string"/>
+      <xsd:element name="hcid" type="xsd:string"/>
+      <!-- Organization status for example, active -->
+      <xsd:element name="status" type="xsd:string"/>
```

Comment:
I think status should have min=0.  Not sure if we pull that from UDDI right now.